exercises/hello-world/hello_world.go

```diff
@@ -10,6 +10,16 @@ package greeting
 // after you have posted this code to the Exercism site -- reviewers
 // will see that your code can't necessarily be expected to pass the
 // current test suite because it was written to an earlier test version.
+//
+// This is a convention done for Exercism exercises in the Go language track,
```

Comment:
Yeah, seems good to make the distinction that it's just something this track uses.